pages.fa/common/echo.md

```diff
@@ -0,0 +1,24 @@
+# echo
+
+> چاپ ورودی داده شده.
+> اطلاعات بیشتر: <https://www.gnu.org/software/coreutils/echo>.
+
+- چاپ پیام ورودی. نکته : استفاده از علامت نقل قول انتخابی است
+
+`echo "{{سلام دنیا}}"`
+
+- چاپ یک پیام حاوی متغییرهای سیستمی
+
+`echo "{{مسیر سیستم من $PATH است.}}"`
+
+- چاپ یک پیام بدون انتقال به خط جدید
+
+`echo -n "{{سلام دنیا}}"`
+
+- افزودن یک پیام به انتهای یک فایل
+
+`echo "{{سلام دنیا}}" >> {{فایل.متنی}}`
+
+- فعال کردن تفسیر کاراکترهای خاص، برای مثال tab
+
+`echo -e "{{ستون 1\tستون 2}}"`
```

Comment:
(Optional) We recently added a new command to the last of the `echo` page.

```
 - Print the exit status of the last executed command (Note: In Windows Command Prompt and PowerShell the equivalent commands are `echo %errorlevel%` and `$lastexitcode` respectively):

`echo $?`
```

Can you translate this too?